dataplane/standalone/apigen/docparser/docparser.go

```diff
@@ -165,10 +171,30 @@ func memberToAttrInfo(enum MemberDef) *Attr {
 	return info
 }
 
-func memberToEnumValueStrings(enum MemberDef) []string {
-	res := []string{}
-	for _, value := range enum.EnumValues {
-		res = append(res, value.Name)
+func memberToEnumValueStrings(enum MemberDef) []*Enum {
+	res := []*Enum{}
+	prev := -1
+	for i, value := range enum.EnumValues {
+		val := prev + 1
```

Comment:
nit: perhaps `next := 0` is slightly better. Perhaps it's just I'm not sure what "prev/next" is just by scanning this.

Reply:
kept the same, but added comment